metaapp/src/main/java/com/sourcepointmeta/app/ui/NewPropertyActivity.java

```diff
@@ -94,12 +92,11 @@ private void removeWebView(View view) {
     private GDPRConsentLib buildConsentLib(Property property, Activity activity) {
         ConsentLibBuilder consentLibBuilder = GDPRConsentLib.newBuilder(property.getAccountID(), property.getProperty(), property.getPropertyID(), property.getPmID(), activity)
                 .setStagingCampaign(property.isStaging())
-                .setMessageTimeOut(15000)
+                .setMessageTimeOut(30000)
```

Comment:
preferably we should recover this default value from the same single static source

Reply:
Done